src/beast/beast/asio/streambuf.h

```diff
@@ -49,11 +51,19 @@ class basic_streambuf
     class element;
 
     using alloc_traits = std::allocator_traits<Allocator>;
```

Comment:
Today the code assumes that `sizeof(Allocator::value_type) == 1`.  In the short term we should `static_assert` that assumption, so that if we break that assumption, we get a compile-time error instead of a run time error.  Longer term we can put a rebind in here to allow any `value_type` if that is desired.